pyproject.toml

```diff
@@ -40,7 +40,7 @@ autoflake = "^2.2.1"
 isort = "^5.13.2"
 markdownify = "^0.11.6"
 tavily-python = "^0.3.9"
-microchain-python = "^0.4.4"
+microchain-python = { path = "../microchain", develop = true }
```

Comment:
TODO: Needs change after Microchain is accepted & released

Reply:
in case you want to merge this before the microchain pr is merged, i guess you'll want to change this to your forked branch with a revision id 😺